apiary.apib

```diff
@@ -79,8 +116,29 @@ Store data as a raw ipfs block.
         ```
 
 ## put [GET /block/put]
-TODO
+Stores input as an IPFS block
+
+ipfs block put is a plumbing command for storing raw ipfs blocks.
+It reads from stdin, and <key> is a base58 encoded multihash.
+
++ Parameters
+    + arg (string, required) - The path to the data to be stored as an IPFS block
```

Comment:
I don't think this works, `file paths` do not work on the http api as far as I know, or at least there is also an alternative way of uploading the data directly